.github/ISSUE_TEMPLATE/vsp-release-plan.md

```diff
@@ -0,0 +1,86 @@
+# Release Plan Template [Draft]
+This Release Plan Template is intended to be used to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necesssary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. 
+
+You do not need to complete a release plan for a product or feature update that does not:
+
+- Add substantial new functionality to the Platform.
+- Impact user flows through tasks.
+
+For your product or feature, make a copy of this template and check it into products/platform/<product-name>.
```

Comment:
deleting a couple things:
- some of the "when to use this" text (assumption: if you're here, you've _already_ been directed to use a release plan)
- directions about copying to a certain directory (nope, moving to this github issue instead)

adding some things:
- italicized "Guidance" section
- directions about filling it out

```suggestion
## Guidance

_This Release Plan Template is intended to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necessary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs.)

_Fill out the **Planning** sections to start, then the **Results** sections as you complete each phase of testing._

---
```